src/main/java/org/engine/process/performance/utils/Utils.java

```diff
@@ -78,15 +74,62 @@ public double standardDeviation(double[] array) {
 	
 	
  
-	public String createCsvFile(double[] rowDataToSourceDiffTimeArray,
+	public String createCsvFileDataInRows(double[] rowDataToSourceDiffTimeArray,
 			double[] sourceToUpdateDiffTimeArray, double[] totalDiffTimeArray,String sourceName) {
 	 
 		StringBuffer output = new StringBuffer(); 
 		output.append("DiffTime between between <"+sourceName+"-row-data> and <"+sourceName+">").append(getLine(rowDataToSourceDiffTimeArray)).append("\n");
 		output.append("DiffTime between <"+sourceName+"> and <update>").append(getLine(sourceToUpdateDiffTimeArray)).append("\n");
 		output.append("Total DiffTime").append(getLine(totalDiffTimeArray)).append("\n");
+		
+		return output.toString();
+			
+ 
+	}
+ 	
+	public String createCsvFileDataInColumns(double[] rowDataToSourceDiffTimeCreateArray,double[] sourceToUpdateDiffTimeCreateArray, double[] totalDiffTimeCreateArray,
```

Comment:
Did you try to work with a standard CSV java library? 
such as  https://commons.apache.org/proper/commons-csv/ ?

Reply:
The main reason to use this kind of libraries is to optimize the read and writing of these files (that we might experience while handling big files)